test/spec/modules/liveIntentIdSystem_spec.js

```diff
@@ -211,7 +211,7 @@ describe('LiveIntentId', function() {
 
   it('should include the LiveConnect identifier and additional Identifiers to resolve', function() {
     const oldCookie = 'a-xxxx--123e4567-e89b-12d3-a456-426655440000'
-    getDataFromLocalStorageStub.withArgs('_li_duid').returns(oldCookie);
+    getCookieStub.withArgs('_lc2_fpi').returns(oldCookie);
```

Comment:
Given the PR description I find it surprising that you'd change these from local storage to cookies - but if it makes sense to you (& the rest of the maintainers) I have no issue with it.

Reply:
@dgirardi The resulting behaviour did not change. A step for building the request was refactored and the dependency on the `_lc2_fpi` cookie being present became more obvious in the test. It was there before too, but it was enough to stub `getDataFromLocalStorage`.